### PR TITLE
Shorten author list #115

### PR DESCRIPTION
**Pull request by frombrest**
#115

### CODE REVIEW

**Comment by nbeloglazov** [comment]
Can you also add a test for the new functionality? In a follow up PR if you want.

**Comment by frombrest** [comment]
image

**Comment by frombrest** [comment]
> Can you also add a test for the new functionality? In a follow up PR if you want.

sure. But let's finish feature first. I mean do you like this implementation or maybe some comments, suggestions? 

`і інш.` - link transfer you to the book details. Is that ok?

**Comment by nbeloglazov** [comment]
Yes, looks good. The only issue is that there is no comma before "і". I'm googling "і інш" and seeing only examples without comma.

**Comment by frombrest** [comment]
image

File: templates/partials/_book_preview.html [hunk 1]
```diff
@@ -26,11 +26,22 @@
...
+            {% endfor %}
+                <a href="{% url 'book-detail-page' book.slug %}" class="text-decoration-none fw-bolder">
+                    і інш.
+                </a>
+        {% else %}
+            {% for author in book.authors.all %}
+                <a href="{% url 'person-detail-page' author.slug %}" class="text-decoration-none fw-bolder">
+                    {{ author.name }}{% if not forloop.last %},{% endif %}
+                </a>
+            {% endfor %}
```
Comment (nbeloglazov) [inline templates/partials/_book_preview.html:43]: Given that this part is identical to the part above. Can we always render first two elements of the list and move "if > 2" check to control rendering only of "і інш"?

Comment (frombrest) [reply templates/partials/_book_preview.html]: I was thinking `|slice:":2"` will throw an exceptions on the lists less than 2 items size, however it works fine.

**Review by nbeloglazov — APPROVED** [review]

**Pull request merged**